ext/fields.go

```diff
@@ -0,0 +1,31 @@
+package ext
+
+import (
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/log"
+)
+
+// These constants define common logFields names recommended for better portability across
+// tracing systems and languages/platforms
+// More info: https://github.com/opentracing/specification/blob/master/semantic_conventions.md
+var (
+	LogEvent   = stringLogName("event")
+	LogMessage = stringLogName("message")
+	LogStack   = stringLogName("stack")
```

Comment:
I think these would make more sense in the `log` pkg

Reply:
also, only the first of these is self-contained Field, the others don't make sense as fields, they are just keys. They would make more sense as functions taking the value string and returning a Field.

Reply:
> I think these would make more sense in the `log` pkg
Sound reasonable, but LogError() depends on opentrace.Span so this result in cyclic dependency between opentrace and log packages.